lib/kitchen/driver/dokken.rb

```diff
@@ -61,6 +61,7 @@ class Dokken < Kitchen::Driver::Base
       default_config :userns_host, false
       default_config :pull_platform_image, true
       default_config :pull_chef_image, true
+      default_config :memory_limit, 0
```

Comment:
Can you add an example of this to the readme so others can discover it as well?

Reply:
Added